content/en/docs/Contribute/doc-con-formatting.md

```diff
@@ -1,180 +0,0 @@
-<!--

```

Comment:
Do we have an alternative to this content somewhere? There are some points that we really should ensure consistency on (e.g. use of inline code for API objects, consistently identifying placeholders in commands, etc.).

Reply:
I didn't want to keep a long list of requisites to contribute to docs for two main reasons:

- Some points are general tech-writing conventions, not Tekton specific.
- Contributors to docs are likely to look at what's already there and create something similar instead of reading the entire guidelines.

I got a bit carried away. I'm summarizing this in a shorter version of this doc and linking from the vendor guidelines.